packages/cli/src/common/filenames.ts

```diff
@@ -1,5 +1,38 @@
 import * as inflected from 'inflected'
+import { logger } from '../services/logger'
+import Brand from './brand'
 
 export function classNameToFileName(name: string): string {
   return inflected.dasherize(inflected.underscore(name))
 }
+
+export async function checkResourceNameIsValid(name: string): Promise<void> {
+  logger.info(Brand.mellancholize('Checking if resource has valid name...'))
```

Comment:
I would't display this message in the console. It only makes sense displaying on errors